internal/app/migrate.go

```diff
@@ -35,9 +36,13 @@ func MigrationError(op string) error {
 	return fmt.Errorf("%w: %s", errMigrate, op)
 }
 
-func Init() error {
+func Init(cfg *config.Config) error {
 	databaseURL, ok := os.LookupEnv("DB_URL")
```

Comment:
looking up from the env should no longer be needed since it is done on load of the config.